### DISCUSSION TITLE
Beatmap listing doesn't load after signing in when initially opened with a sign in placeholder

### DISCUSSION THREAD

**Discussion by Joehuu** [ppy/osu#16957] [Q&A] [answered]
Steps to reproduce:
- Start osu!
- Sign out if not already
- Open beatmap listing overlay
- Sign in
- Don't type anything

The overlay will load indefinitely, and only typing will fix the state. Video is cropped to hide password length.

https://user-images.githubusercontent.com/35318437/155066503-a23b78f1-2b6c-48ff-8bb9-82f555b314c3.mov

**Comment by peppy** [comment, marked as answer]
Can confirm I've also experienced this. Pretty sure it worked at some point, so a likely regression.

Tracking via https://github.com/ppy/osu/issues/16958.
